R/checkers.R

```diff
@@ -0,0 +1,84 @@
+#' Checks whether <epidist> or <epiparam> object contains a distribution and
+#' parameters for that distribution.
+#'
+#' @description If the <epidist> object or a row in the <epiparam> data frame
+#' is missing either a probability distribution or parameters for the
+#' probability distribution, `is_parameterised` returns `FALSE`, otherwise
+#' it returns `TRUE`.
+#'
+#' @param x An `epidist` or `epiparam` object.
+#' @param ... [`dots`] not used, extra arguments supplied will cause a warning.
+#'
+#' @return A boolean logical for <epidist> or vector of boolean for each entry
+#' in the <epiparam>.
+#' @export
```

Comment:
Since this function is exported, would it be possible to add in the description or in the examples a typical use case for users? At the moment, I can only think about internal use cases.